plugin.video.uzg/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.uzg" name="Uitzendinggemist (NPO)" provider-name="Bas Magré (Opvolger)" version="1.3.0">
+<addon id="plugin.video.uzg" name="Uitzendinggemist (NPO)" provider-name="Bas Magré (Opvolger)" version="1.3.1">
   <requires>
     <import addon="xbmc.python" version="2.1.0" />
```

Comment:
Please consider to update the python API version on the next update to match gotham. See kodi.wiki/view/addon.xml